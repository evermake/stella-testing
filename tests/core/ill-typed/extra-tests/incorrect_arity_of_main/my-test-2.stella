language core;

extend with #nullary-functions;

fn Bool::not(b : Bool) -> Bool {
  return
    if b then false else true ;
}

fn main() -> Bool {
  return Bool::not(true);
}
