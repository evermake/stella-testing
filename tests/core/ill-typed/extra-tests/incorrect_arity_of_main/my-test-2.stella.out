TYPE ERROR:
Type Error Tag: [ERROR_INCORRECT_ARITY_OF_MAIN]
main function must have exactly one parameter